tox.ini

```diff
@@ -1,29 +1,28 @@
 [tox]
-envlist = py27,py27-simplejson,py34
+envlist = {py27,py35,py36}-{default,simplejson},pre-commit,cover,docs
```

Comment:
Do we really need to test all of those Python versions with simplejson? Not knowing why this was introduced, I'd rather vote for dropping it, or are there known incompatibilities in recent versions?